eos-tech-support/eos-convert-system-qa

```diff
@@ -154,8 +154,18 @@ if $OSTREE || $APPS; then
             --set=branches="${new_branch};"
 
         # Check for an update every time the updater runs
-        # rather than once every two weeks
-	sed -i 's/IntervalDays=14/IntervalDays=0/' /etc/eos-updater.conf
+        # rather than once every two weeks. eos-updater.conf is the old
+        # location of the config file.
+        if [ -f /etc/eos-updater.conf ]; then
```

Comment:
`/etc/eos-updater/eos-autoupdater.conf` will not exist; `/usr/share/eos-updater/eos-autoupdater.conf` will exist.